flake.nix

```diff
@@ -40,9 +40,7 @@
 
         overlays = import ./overlays;
 
-        nixosModules = (import ./modules) // {
-          soxin = import ./modules/soxin.nix;
-        };
+        nixosModules.soxin = import ./modules/soxin.nix;
```

Comment:
Does that mean we don't want to expose individual modules? For example, I would have liked to reuse the bluetooth module for a work project. I know I can import everything and just turn the bluetooth on, but that means having blind trust into the upstream project that nothing is set by default, and it also means that upgrading requires reviewing every single change to make sure of that, instead of only reviewing the changes to the list of modules (to make sure the bluetooth module hasn't added some other files) and to the bluetooth module.

Reply:
I have not thought of that use case. Do you think they would individually fit the top-level nixosModules attribute or should it go in a sub-attribute?

Reply:
I'm not sure flakes allow for modules to be nested in `nixosModules`.

Reply:
Let me add it back as it was then.

Reply:
done